### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by silverrain**
Updated readme with development instructions.

### CODE REVIEW

**Comment by ralphbean** [comment]
Cool.  This is merged in in 56ff01648120d852ea7fe60a3095ef15d8aefe62 and 9ce9ceff191cc161e9f96f4da5959de23b9be447

File: README.rst [hunk 1]
```diff
@@ -9,37 +9,96 @@ sqlalchemy database.
...
+The source code is on github. 
+
+Create a working folder to hold the source files::
+
+    $ mkdir source
+    $ cd source
+
+Get fedmsg::
+
+    $ git clone https://github.com/ralphbean/fedmsg.git
```
Comment (sontek) [inline README.rst:45]: We should document that they should run git checkout develop after this line to make sure they are working in the develop branch and not off of master, same for line #49

File: README.rst [hunk 2]
```diff
@@ -9,37 +9,96 @@ sqlalchemy database.
...
+
+In a third, emit a message, which gets picked up by the relay, rebroadcasted, consumed by datanommer, and stuffed into /tmp/datanommer.db::
+
+    $ echo "this is a test" | fedmsg-logger
 
-::
+Use datanommer's clumsy CLI tools to inspect the DB. Was the message stored?::
 
     $ /usr/bin/datanommer-stats
     $ /usr/bin/datanommer-dump
```
Comment (sontek) [inline README.rst:104]: These commands wont be in /usr/bin if they are working within the virtualenv.   So they should be something like ../bin/datanommer-stats

File: README.rst [hunk 3]
```diff
@@ -9,37 +9,96 @@ sqlalchemy database.
...
+
+    (source)$ fedmsg-relay
+
+In another, type::
+
+    (source)$ fedmsg-hub
+
+In a third, emit a message, which gets picked up by the relay, rebroadcasted, consumed by datanommer, and stuffed into /tmp/datanommer.db::
+
+    $ echo "this is a test" | fedmsg-logger
```
Comment (sontek) [inline README.rst:99]: Do you think its important to show them that the data is stored in the SQL database that they created in line #80?   So them how to run sqlite3  and query the logged message?

File: README.rst (hunk 2 above)
Comment (ralphbean) [inline README.rst:104]: Ah, that's correct.  This is the most pressing change.

**Pull request closed without merging**